src/Chakra.res

```diff
@@ -55,11 +55,11 @@ module CloseButton = Chakra__CloseButton
 module Code = Chakra__Code
 module Divider = Chakra__Divider
 module Kbd = Chakra__Kbd
+module Tag = Chakra__Tag
```

Comment:
You may be forgot or missing module declaration for `TagCloseButton`, `TagLabel`, & `TagLeftIcon`. 

So, you need to add ```rescript
module TagCloseButton = Chakra__TagCloseButton
module TagCloseButton = Chakra__TagLabel
module TagCloseButton = Chakra__TagLeftIcon
```

Reply:
Solved by 8d9b3a9f377943e6b6ae857ef8aaa818aef880fa commit